content/cosign-verify-bundles.md

```diff
@@ -0,0 +1,110 @@
++++
+title = "cosign Verification of npm Provenance, GitHub Artifact Attestations, and Homebrew Provenance"
+author = "Zach Steindler (GitHub)"
+date = "2024-08-09"
```

Comment:
you might want to update this before publishing

Reply:
I feel you, but that will again dismiss all the reviews. Is this something a maintainer can do before landing? Or should I update it and hope we get timely re-reviews?

Reply:
I think you're fine. I don't see any outstanding comments, anyway (mine is a suggestion :))

Reply:
if you have 3 TSC LGTMs, i'm happy to force merge for a trivial fix like setting a date.